backend/src/main/java/gov/cdc/usds/simplereport/api/model/filerow/PatientUploadRow.java

```diff
@@ -45,35 +45,72 @@ public class PatientUploadRow implements FileRow {
   final ValueOrError role;
   final ValueOrError email;
 
+  static final String FIRST_NAME = "first_name";
+  static final String LAST_NAME = "last_name";
+  static final String RACE_FIELD = "race";
```

Comment:
⚠️ sonar didn't like that `RACE` is too similar to `race` so had to add `_FIELD` to a couple constants